.travis.yml

```diff
@@ -5,11 +5,9 @@ notifications:
   email: false
 
 python:
+  - '3.8'
   - '3.7'
   - '3.6'
```

Comment:
Do we still need this version? Is it used somewhere or just for compatibility?

Reply:
Just keeping support for it as a [non-trivial amount of users are still downloading it for 3.6](https://pypistats.org/packages/apiron), and because 3.6 remains officially supported until ~EOY 2021.

Reply:
Never mind, it's in the description :p